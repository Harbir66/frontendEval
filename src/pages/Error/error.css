.main{
  display: flex;
  flex-direction: column;
  min-height: 100vh;
  justify-content: center;
}
.errorContainer {
  flex:1;
  margin-top: 33vh;
  font-size: larger;
}