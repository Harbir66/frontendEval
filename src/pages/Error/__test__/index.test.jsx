import React from 'react';
import { render } from '@testing-library/react';
import Error from '../index';

describe('Error Page', () => {
  it('should render', () => {
    const { asFragment } = render(<Error />);
    expect(asFragment()).toMatchSnapshot();
  });
});
