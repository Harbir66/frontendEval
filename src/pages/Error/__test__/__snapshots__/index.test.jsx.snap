// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Page should render 1`] = `
<DocumentFragment>
  <div
    class="main"
  >
    <header
      class="padding"
    >
      EVENTIFY
    </header>
    <div
      class="errorContainer"
    >
      <p>
        Something went wrong!
      </p>
    </div>
    <footer
      class="padding"
    >
      <p>
        THEMES
      </p>
      <div
        class="themes"
      />
    </footer>
  </div>
</DocumentFragment>
`;
